### PR TITLE
increased max header and body size to better support large requests and a reverse proxy

### PR DESCRIPTION
**Pull request by PaulKobow7536**
We need a larger body size, because some of our requests are larger than vertex default
We need larger Header size to support large Authentication headers in a simple reverse proxy

### CODE REVIEW

**Comment by jonrosner** [comment]
Thanks for your MR. I won't add this as the default behavior but adding an option to increase the max body and header size is fine for me. So you can add a new environment variable. Check `Main.kt`.

**Comment by PaulKobow7536** [comment]
Hey i added the requested changes. Would you mind having a look?

File: server/src/main/kotlin/com/espero/yaade/Main.kt [hunk 1]
```diff
@@ -17,6 +19,8 @@ val ADMIN_USERNAME: String = System.getenv("YAADE_ADMIN_USERNAME") ?: ""
 val BASE_PATH: String = System.getenv("YAADE_BASE_PATH") ?: ""
 val FILE_STORAGE_PATH: String = System.getenv("YAADE_FILE_STORAGE_PATH") ?: "./app/data/files"
 val SCRIPT_RUNNER_TIMEOUT: Long = System.getenv("YAADE_SCRIPT_RUNNER_TIMEOUT")?.toLong() ?: 30_000
+val VERTX_BODY_LIMIT: Long = System.getenv("VERTX_BODY_LIMIT")?.toLong() ?: DEFAULT_BODY_LIMIT
```
Comment (jonrosner) [inline server/src/main/kotlin/com/espero/yaade/Main.kt:22]: rename them to YAADE_VERTX_BODY_LIMIT

**Comment by jonrosner** [comment]
LGTM

**Comment by jonrosner** [comment]
I deployed the new container to :nightly. Please check if it works for you.

**Pull request merged**